includes/class-wc-product-variable.php

```diff
@@ -282,12 +282,11 @@ public function get_virtual( $context = 'view' ) {
 	/**
 	 * Get an array of available variations for the current product.
 	 *
-	 * @param bool $render_variations Prepares a data array for each variant for output in the add to cart form. Pass `false` to only return the available variations as objects.
-	 * @param bool $return_array_of_data If true, return an array of data for the variation; if false, return a WC_Product_Variation object.
+	 * @param bool $return Optional. The format to return the results in. Can be 'array' to return an array of variation data or 'objects' for the product objects. Default 'array'.
 	 *
-	 * @return array|WC_Product_Variation
+	 * @return array[]|WC_Product_Variation[]
 	 */
-	public function get_available_variations( $render_variations = true, $return_array_of_data = true ) {
```

Comment:
I also replaced the two boolean arguments with a string literal argument to be consistent with the coding standard here: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#self-explanatory-flag-values-for-function-arguments

Reply:
Nice update based on WP coding standards 👍